pwvlt-cli/src/main.rs

```diff
@@ -94,36 +112,32 @@ fn handle_args(args: ArgMatches) -> Result<(), Error> {
     }
 }
 
-fn handle_store_errors(err: PassStoreError) {
+fn handle_backend_errors(err: PwvltError) {
     match err {
-        PassStoreError::Keyring(e) => error!(
+        PwvltError::Keyring(e) => error!(
             "An error occurred while accessing the Keyring backend: {}",
             e
         ),
-        PassStoreError::GeneralError(e) => error!("An internal error occurred: {}", e),
-        PassStoreError::IoError(e) => error!("An internal IO error occurred: {}", e),
-        PassStoreError::NitrokeyError(e) => error!(
+        PwvltError::Io(e) => error!("An internal IO error occurred: {}", e),
+        PwvltError::Nitrokey(e) => error!(
             "An error occurred while accessing the Nitrokey backend: {}",
             e
         ),
-        PassStoreError::PasswordGenerationError(e) => error!(
+        PwvltError::PasswordGeneration(e) => error!(
             "An error occurred while generating a random password: {}",
             e
         ),
-        PassStoreError::PasswordNotFound => error!("No password could be found!"),
-        PassStoreError::SkipError => unimplemented!("SkipError"),
-        PassStoreError::LockedError => {
-            error!("An error occurred while unlocking the Nitrokey backend")
-        }
-        PassStoreError::Utf8(_) => error!("Failed to parse a password as Utf8"),
+        PwvltError::PasswordNotFound => error!("No password could be found!"),
+        PwvltError::Skip => unimplemented!("SkipError"),
+        PwvltError::Utf8(_) => error!("Failed to parse a password as Utf8"),
     }
 }
 
 fn main() {
     let matches = App::new("Password Vault")
         .version("1.0")
         .author("Robert B. <bartlensky.robert@gmail.com>")
-        .about("Stores passwords on the local keyring or on a Nitrokey.")
+        .about("Backends passwords on the local keyring or on a Nitrokey.")
```

Comment:
Substitution error: this should still say 'Stores'.

Reply:
Done!